internal/service/privatelinkendpointservicedatafederationonlinearchive/data_source_privatelink_endpoint_service_data_federation_online_archive_test.go

```diff
@@ -13,15 +14,19 @@ var (
 )
 
 func TestAccNetworkPrivatelinkEndpointServiceDataFederationOnlineArchiveDS_basic(t *testing.T) {
-	// Skip because private endpoints are deleted daily from dev environment
-	acc.SkipTestForCI(t)
+	var (
+		projectID               = acc.ProjectIDGlobal(t)
```

Comment:
unrelated to this PR: are we planning to document and explain how to re-use the different level of projects?

Reply:
at least i was planning to describe the list of shared projects and what they're used for, but yeah we can add more help about how to decide what level of reuse to do, let me think more about how to document that